feature-libs/cart/base/root/models/cart.model.ts

```diff
@@ -118,6 +118,7 @@ export interface CartItemComponentOptions {
   displayAddToCart?: boolean;
   addToCartString?: string;
   cartType?: CartType;
+  disableItemLink?: boolean;
```

Comment:
introducing optional new toggle to active/unactive anchor tag on item title.
context: we don't want user to navigate to pdp.